packages/constants/src/assets.ts

```diff
@@ -6,6 +6,14 @@ export const localAssets: Metadata[] = LocalAssetRegistry.map(a =>
   Metadata.fromJson(a as JsonValue),
 );
 
+export const NUMERAIRE_DENOMS: string[] = ['test_usd', 'usdc'];
+export const NUMERAIRES: Metadata[] = localAssets.filter(m => NUMERAIRE_DENOMS.includes(m.display));
```

Comment:
I assume we will have two different local assets registy for testnet and mainnet. And we will have to revise the definition of `NUMERAIRES`